### ISSUE TITLE
Consider moving the project to `github.com/coredhcp/dhcp`

### ISSUE THREAD

**Issue by pmazzini** [insomniacslk/dhcp#243]
I am only opening this to discuss whether we should move the project to `github.com/coredhcp/dhcp`.

The cons are that we already have the issues, stars and metrics in the current repo.

**Comment by stapelberg** [comment]
> The cons are that we already have the issues, stars and metrics in the current repo.

AFAIK, you can move a repository without losing any of these.

**Comment by insomniacslk** [comment]
yeah, you can move a repo and retain commit history, issues, etc, it would be just like a rename. Also, there is automatic redirection from the old to the new repo, so even importing code via `go get` should keep working with the old import path.
However CoreDHCP is not yet mature enough, and I'd rather not move it until then.

**Comment by pmazzini** [comment]
Inspired by the `go-ping` and `go-zookeeper` orgs, I registered the `go-dhcp` one. I am happy to transfer/add owners to it.

**Comment by hugelgupf** [comment]
I'm for it.

**Comment by pmazzini** [comment]
Reviving this, would it make sense to move it to an org?

**Comment by insomniacslk** [comment]
No objections, I would use the existing coredhcp org though
